core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -655,12 +660,14 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
   private def deleteWorkspaceInternal(workspaceContext: Workspace,
                                       maybeMcWorkspace: Option[WorkspaceDescription],
                                       parentContext: RawlsRequestContext
-  ): Future[WorkspaceDeletionResult] =
-    for {
-      _ <- Applicative[Future].unlessA(isAzureMcWorkspace(maybeMcWorkspace))(
-        assertNoGoogleChildrenBlockingWorkspaceDeletion(workspaceContext)
+  ): Future[WorkspaceDeletionResult] = {
+    if (isAzureMcWorkspace(maybeMcWorkspace)) {
```

Comment:
This code path is not supported anymore for MC workspaces, it's all flowing through the MC service [here](https://github.com/broadinstitute/rawls/blob/develop/core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala#L114) which prevents this code from being called w/mc workspaces.